packages/trader/src/Modules/CFD/Containers/compare-accounts-modal.tsx

```diff
@@ -466,34 +577,41 @@ const CFDCompareAccountHint = ({ platform, show_risk_message, landing_companies,
     );
 };
 
-const ModalContent = ({ landing_companies, is_logged_in, platform, show_eu_related, residence, is_eu, is_uk }) => {
-    const [cols, setCols] = React.useState([]);
+const ModalContent = ({
+    landing_companies,
+    is_logged_in,
+    platform,
+    show_eu_related,
+    residence,
+    is_eu,
+    is_uk,
+}: TModalContentProps) => {
+    const [cols, setCols] = React.useState<Array<{ [key: string]: string | React.ReactNode | undefined }>>([]);
```

Comment:
```suggestion
    const [cols, setCols] = React.useState<Array<Record<string, string | React.ReactNode | undefined>>>([]);
```
https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeys-type
Using `Record` utility type will improve readability.